eng/SourceBuild.Tarball.targets

```diff
@@ -297,20 +297,21 @@
   </Target>
 
   <Target Name="AddTarballLeakDetection"
-          Condition="'$(IncludeLeakDetection)' == 'true'">
+          Condition="'$(IncludeLeakDetection)' == 'true'"
+          DependsOnTargets="CopyTarballFiles">
```

Comment:
The targets in this file should probably get order via dependenices, not the big list at the top... but I think adding this one for now is a good start.